terraform/instances.tf

```diff
@@ -565,24 +589,4 @@ data "template_file" "sumo_sources_syslog" {
   }
 }
 
-data "template_file" "windows_worker_user_data" {
-  template = file("${path.module}/templates/windows_worker_user_data")
-
-  vars = {
-    environment = var.env
-    password    = var.admin_password
-    flags       = "--no-color --auto-update --peer ${join(" ", var.peers)} --channel ${var.sup_release_channel} --listen-gossip 0.0.0.0:${var.gossip_listen_port} --listen-http 0.0.0.0:${var.http_listen_port}"
-    bldr_url    = var.bldr_url
-    channel     = var.release_channel
-  }
-}
-
-data "template_file" "linux2_init" {
-  template = file("${path.module}/templates/hab-sup.init")
-
-  vars = {
-    flags     = "--auto-update --peer ${join(" ", var.peers)} --channel ${var.sup_release_channel} --listen-gossip 0.0.0.0:${var.gossip_listen_port} --listen-http 0.0.0.0:${var.http_listen_port}"
-    log_level = var.log_level
-  }
 }
```

Comment:
There appears to be an orphaned `}` at the very end of `instances.tf`

```
11:56 $ terraform validate

Error: Argument or block definition required

  on instances.tf line 592:
 592: }
```

Reply:
Good catch @jeremymv2 

:thinking: I think I may take this opportunity to add a `terraform validate` stage to the pipeline.

Reply:
Fixed in https://github.com/habitat-sh/builder/pull/1252/commits/892356a8e0e89577f7041d23010a7d1eb4950faf